server/main.go

```diff
@@ -1,3 +1,4 @@
+// Command server is the Instant Share server. It is responsible for accepting uploads and serving those files.
```

Comment:
This is the standard way to document packages in Go.

If the package name is main, the package comment should begin with "// Command {{base}} ..." where base is the parent folder name (and therefore the default name of the produced binary).

If the package name is not main, the package comment should begin with "// Package {{name}} ..." where name is the package name.

Source: https://github.com/golang/go/wiki/CodeReviewComments#package-comments

Actually, it seems it only applies for non-main packages, see [here](https://github.com/golang/lint/blob/7b7f4364ff76043e6c3610281525fabc0d90f0e4/lint.go#L399-L402). So we can make this nicer. Suggestions?